query/office365_domain_sharing_restriction_mode.sql

```diff
@@ -0,0 +1,16 @@
+-- Ensure document sharing is being controlled by domains with whitelist or blacklist
+select
+  -- Required columns
+  tenant_id as resource,
+  case
+    when sharing_domain_restriction_mode  = 'allowList' or sharing_domain_restriction_mode = 'blockList' then 'ok'
+    else 'alarm'
+  end as status,
+  case
+     when sharing_domain_restriction_mode  = 'allowList' or sharing_domain_restriction_mode = 'blockList' then 'document sharing is being controlled by domains with blacklist or whitelist.'
+     else 'document sharing is not being controlled by domains with blacklist or whitelist.'
```

Comment:
```suggestion
     else 'document sharing controlled by domains with blacklist or whitelist.'
```
same way in else conditions

Reply:
yes fixed